openshift-install-powervs

```diff
@@ -831,70 +841,82 @@ function variables_nodes {
     no_of_bastion=1
   fi
 
-  question "Do you want to use the default configuration for all the cluster nodes?" "yes no"
-  if [ "${value}" == "yes" ]; then
-    {
-      echo "bastion = {memory = \"16\", processors = \"1\", \"count\" = $no_of_bastion}"
-      echo "bootstrap = {memory = \"32\", processors = \"0.5\", \"count\" = 1}"
-      echo "master = {memory = \"32\", processors = \"0.5\", \"count\" = 3}"
-      echo "worker = {memory = \"32\", processors = \"0.5\", \"count\" = 2}"
-    } >> "$VAR_TEMPLATE"
+  if [ ! -s $flavortype ]; then
+    cat automation/compute-vars/$flavortype.tfvars >> "$VAR_TEMPLATE"
     return 0
   fi
-
-  # Bastion node config
-  question "Do you want to use the default configuration for bastion node? (memory(GB)=16 processors=1)" "yes no"
-  if [ "${value}" == "yes" ]; then
-    echo "bastion = {memory = \"16\", processors = \"1\", \"count\" = $no_of_bastion}" >> "$VAR_TEMPLATE"
+  flavortype=$(ls automation/compute-vars/ && echo CUSTOM)
+  question "Select the flavor for the cluster nodes:" "${flavortype}" yes
+  if [ "${value}" != "CUSTOM" ]; then
+    vars="-var-file ../$varfile -var-file ../automation/compute-vars/$value" >> "$VAR_TEMPLATE"
+    cat automation/compute-vars/$value >> "$VAR_TEMPLATE"
```

Comment:
Above lines should go in else part of previous condition.

All the below lines have no change so no need reflect it in this change set.

Reply:
Modified